src/state/CallViewModel.ts

```diff
@@ -496,6 +507,7 @@ export class CallViewModel extends ViewModel {
       }
       return displaynameMap;
     }),
+    this.scope.state(),
```

Comment:
This, I take it, is the line that actually fixes things

Reply:
Yes, this is the important step.